holidayextras/ui-toolkit
# Improve justified container for older browsers including safari

#### What does this PR do? (please provide any background)
Fixes the justified container component for older browsers including safari - https://github.com/holidayextras/ui-toolkit/issues/55

#### What tests does this PR have?
None - i've actually removed one of the tests which checks for inline styles

#### How can this be tested?
run `grunt docs` and check justified container in safari.
compare www.ui-toolkit.com with your local version on safari. you will see that the justified conainter now works fully.

#### Screenshots / Screencast
<img width="861" alt="screen shot 2015-08-14 at 12 57 01" src="https://cloud.githubusercontent.com/assets/778942/9273354/fc524384-4283-11e5-89e8-fb538e7f65a9.png">
<img width="852" alt="screen shot 2015-08-14 at 12 56 01" src="https://cloud.githubusercontent.com/assets/778942/9273356/ff07beba-4283-11e5-8df1-3461f968e3fe.png">


#### What gif best describes how you feel about this work?
![I'm too flexy](http://media.giphy.com/media/1387MhQBPBqZb2/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
